src/System.Linq.Expressions/tests/CompilerTests.cs

```diff
@@ -115,6 +115,97 @@ public static void EmitConstantsToIL_LiftedToClosure()
             VerifyEmitConstantsToIL((DateTime?)DateTime.Now, 1);
         }
 
+        [Fact]
+        public static void VerifyIL_Simple()
+        {
+            Expression<Func<int>> f = () => Math.Abs(42);
+
+            VerifyIL(f,
+                @".method int32 ::lambda_method(class [System.Linq.Expressions]System.Runtime.CompilerServices.Closure)
+                  {
+                    IL_0000: ldc.i4.s   42
+                    IL_0002: call       int32 class [System.Private.CoreLib]System.Math::Abs(int32)
```

Comment:
@stephentoub - I've added the changes I've made to my original fork of the visualizer here as well, to produce better and more familiar IL syntax. This adds the assembly name on types though. I'm wondering whether we can expect to see a lot of types moving around going forward, causing breaks here? Most tests will likely use members on very basic functionality or on a types defined in the test assembly. If we're concerned about it being too brittle, I can introduce/use a printing mode where the assembly name is omitted.